test/karma/detected.js

```diff
@@ -29,6 +29,7 @@ module.exports = function(config) {
         postDetection: function(availableBrowsers) {
           var safariIndex = availableBrowsers.indexOf('Safari');
           if(safariIndex !== -1) {
+            console.log("Not running safari it is/was broken");
```

Comment:
Yeah, that makes sense. It's not where we *want* to be though, so it would be good to at least keep track of it.